src/account/accounts.module.ts

```diff
@@ -1,13 +1,11 @@
 import { Module } from '@nestjs/common';
 
+import { DbModule } from '../core/db/db.module';
 import { AccountsService } from './accounts.service';
-import { AccountsController } from './accounts.controller';
-import { DbModule } from 'src/core/db/db.module';
 import { accountsProviders } from './accounts.providers';
 
 @Module({
   imports: [DbModule],
   providers: [AccountsService, ...accountsProviders],
-  controllers: [AccountsController],
```

Comment:
We are completely removing this as all the routes would be handled in the auth controller. So, instead of injecting auth module/auth service into the accounts module, we have to actually inject accounts into the auth. We cannot do the other way around.